frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -172,13 +233,21 @@ const Board: React.FC<{ board: string; pollInterval: number; timespan: number }>
   );
 };
 
-const MonitoringDashboardsPage: React.FC<{}> = () => {
-  const [pollInterval, setPollInterval] = React.useState(
+const MonitoringDashboardsPage_: React.FC<MonitoringDashboardsPageProps> = ({
+  clearVariables,
+  patchVariable,
+}) => {
+  const [pollInterval, setPollInterval] = React.useState<number>(
     parsePrometheusDuration(defaultPollInterval),
   );
-  const [timespan, setTimespan] = React.useState(parsePrometheusDuration(defaultTimespan));
+  const [timespan, setTimespan] = React.useState<number>(parsePrometheusDuration(defaultTimespan));
```

Comment:
If `parsePrometheusDuration` has a return type, the type should be inferred

Reply:
Thanks, removed by #3965